src/DotNetLightning.Core/Crypto/Sphinx.fs

```diff
@@ -133,7 +133,7 @@ module Sphinx =
                     let payload = bin.[0..PayloadLength - 1]
                     let hmac = bin.[PayloadLength .. PayloadLength + MacLength - 1] |> uint256
                     let nextRouteInfo = bin.[PayloadLength + MacLength..]
-                    let nextPubKey = blind(pk) (computeBlindingFactor(pk) (new Key(ss)))
+                    let nextPubKey = blind(publicKey) (computeBlindingFactor(publicKey) (new Key(ss)))
```

Comment:
hey, should we save `new Key(ss)` in a variable first, so that you can use `use`?

Reply:
Yeah that might be slightly better for the performance.

Reply:
Done

Reply:
@aarani "separate variable defintion" is not a good name for that commit message because it gives the dev the impression that it's a refactoring; however, as we have said, disposing it early is a performance matter, not a refactoring. I would use the word micro-optimization in the commit message.